test/testing.go

```diff
@@ -0,0 +1,16 @@
+package test
+
+import (
+	"os"
+	"path"
+	"runtime"
+)
+
+func init() {
```

Comment:
Just my 2cents, I actually really like this, it is pretty idiomatic, the `init` name is needed since it is the golang trick to execute code when the package is imported somewhere

Reply:
the `init` function is a golang feature to initialize modules by importing it like `_ "the-module"`
Changing the name would lose the trigger. I'll add some documentation :wink:

Reply:
FTR: the root folder in test would be `.` the root of `trento`, not `./test`